std/range/package.d

```diff
@@ -7250,7 +7250,7 @@ if (isInputRange!Range)
     }
 
     // Assertion only.
-    private void dbgVerifySorted()
+    private auto dbgVerifySorted()
```

Comment:
@atilaneves In 2.068, auto functions function attributes are inferred http://dlang.org/changelog/2.068.0.html#attribinference3